packages/components/input/src/input.vue

```diff
@@ -443,7 +443,35 @@ watch(
   }
 )
 
+// Get the widths of 'suffix' and 'prefix' to set the padding property of the input
+// https://github.com/element-plus/element-plus/issues/6464
+const ElInnerSuffix = ref(null)
+const ElInnerPrefix = ref(null)
+const inputStyleInner = ref({})
```

Comment:
Please add type annotation.
```suggestion
const inputStyleInner = ref<CSSProperties>({})
```